doc/user-guide/user-guide.rst

```diff
@@ -0,0 +1,39 @@
+.. _User_Guide:
+
+#######################
+Electrometer User Guide
+#######################
+
+The electrometer is a Keithley Spectrograph that measures the amount of electrons in proximity with a sensor.
+It communicates over RS-232 serial cable.
+The computer that it is connected to uses a RS-232 to USB type A cable.
+
```

Comment:
Please explain the output of the CSC (the fits file) and how one would access the data.